randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1258,12 +1259,12 @@ Extra - room_id: [40]
   * L0 Hatch to Clogged Cavern/Door to Menu Corridor
   * Extra - door_idx: (47,)
   > Door to Menu Save Room
-      Trivial
+      Combat (Beginner)
```

Comment:
1) This change is likely unnecesary as accessing the save is never going to do anything logically (this could be useful in the future hypothetical entrance rando)
2) This makes it logically impossible for this path to be crossed trickless.
If you feel strongly about this add combat beginner OR beam-weak enemies.
Same comment for below.

Reply:
Isn't killing the menus in that room really trivial?

Reply:
Yeah that's what I also meant by point 1.

Reply:
It probably is trivial. The thought here is more so with potential entrance rando, or starting in that save room. It could easily be moved back to trivial or do what Jeff suggests and make this "beam-weak enemies." I figured it's better to be thorough, at the very least. I'm not sure if anyone _would_ struggle in this room, but I also imagine the number is also not 0.